app/views/content/is-teaching-right-for-me/get-school-experience.md

```diff
@@ -45,6 +49,10 @@ During your experience, you’ll get to do things like:
 - speak to teachers and meet pupils
 - learn more about teacher training - including the application and interview process
 
+<br />
+$q-Zainab$
+<br />
```

Comment:
You shouldn't need this break after the quote? It's creating a lot of white space

Reply:
No problem. I'll delete the break. I copied it from the quote on another page so I don't know if that may need the spacing removing from it too?